pkgs/tools/misc/gay/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, python39Packages, fetchFromGitHub, runCommand, callPackage }:
```

`runCommand` doesn't seem to be used:
```suggestion
{ lib, python39Packages, fetchFromGitHub, callPackage }:
```